File: Java/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/exerciciosFaccat/Exercicio30.java
package exerciciosFaccat;

import java.util.Scanner;

/*30) Ler 3 valores (considere que não serão informados valores iguais) e escrevê-los em ordem
crescente. */

public class Exercicio30 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
				
				int valorA;
				int valorB;
				int valorC;
		
				System.out.println("Digite o valor A:");
				valorA = sc.nextInt();
				System.out.println("Digite o valor B:");
				valorB = sc.nextInt();
				System.out.println("Digite o valor C:");
				valorC = sc.nextInt();

				if ((valorA <= valorB) && (valorA <= valorC) && (valorB <= valorC)) {
					System.out.println(valorA+ " "+ valorB+ " "+ valorC);
				}

				if ((valorA <= valorB) && (valorA <= valorC) && (valorC <= valorB))  {
					System.out.println(valorA+ " "+ valorC+ " "+ valorB);
				}

				if ((valorB <= valorA) && (valorB <= valorC) && (valorA <= valorC)) {
					System.out.println(valorB+ " "+ valorA+ " "+ valorC);
				}

				if ((valorB <= valorA) && (valorB <= valorC) && (valorC <= valorA)) {
					System.out.println(valorB+ " "+ valorC+ " "+ valorA);
				}

				if ((valorC <= valorA) && (valorC <= valorB) && (valorA <= valorB)) {
					System.out.println(valorC+ " "+ valorA+ " "+ valorB);
				}

				if ((valorC <= valorA) && (valorC <= valorB) && (valorB <= valorA)) {
					System.out.println(valorC+ " "+ valorB+ " "+ valorA);
				}
		
		sc.close();
	}

}

File: LogicaDeProgramacao(JavaScript)/Exercicios 5 até o 39 Apostila Faccat/js/exercicio30.js
/*30) Ler 3 valores (considere que não serão informados valores iguais) e escrevê-los em ordem
// crescente. */


var valorA = parseInt(prompt("Digite o valor A:"))
var valorB = parseInt(prompt("Digite o valor B:"))
var valorC = parseInt(prompt("Digite o valor C:"))

if ((valorA <= ValorB) && (valorA <= ValorC) && (valorB <= ValorC)) {
    alert(valorA+ " "+ valorB+ " "+ valorC)
}

if ((valorA <= ValorB) && (valorA <= ValorC) && (valorC <= ValorB))  {
    alert(valorA+ " "+ valorC+ " "+ valorB)
}

if ((valorB <= ValorA) && (valorB <= ValorC) && (valorA <= ValorC)) {
    alert(valorB+ " "+ valorA+ " "+ valorC)
}

if ((valorB <= ValorA) && (valorB <= ValorC) && (valorC <= ValorA)) {
    alert(valorB+ " "+ valorC+ " "+ valorA)
}

if ((valorC <= ValorA) && (valorC <= ValorB) && (valorA <= ValorB)) {
    alert(valorC+ " "+ valorA+ " "+ valorB)
}

if ((valorC <= ValorA) && (valorC <= ValorB) && (valorB <= ValorA)) {
    alert(valorC+ " "+ valorB+ " "+ valorA)
}








